tests/load-tests/run.sh

```diff
@@ -1,3 +1,3 @@
-export DOCKER_CONFIG_JSON=ewogICJhdXRocyI6IHsKICAgICJxdWF5LmlvIjogewogICAgICAiYXV0aCI6ICJiWE5oZDI5dlpEcHJjRVJ6TjNOWWNGWnFUa1I1VGtZMFRUZ3hUR3RMZEdWMmVqVklNSEYyZFc1T2VGVkxVWGRNSzJWWFZIWm9PV1EzUVZaaVQydDRLM2RGV2pWTVpWSXciLAogICAgICAiZW1haWwiOiAiIgogICAgfQogIH0KfQ==
+export DOCKER_CONFIG_JSON=<Insert_token_here>
```

Comment:
Better to check if DOCKER_CONFIG_JSON env exists and if not fail the script saying that DOCKER_CONFIG_JSON env need to be defined